### PR TITLE
Add Warning For Pulsar Page

### PR DESCRIPTION
**Pull request by ImmaZoni**
This PR introduces a series of warning messages across all Pulsar-related pages. These warnings explicitly inform users of the current operational issues with Pulsar, emphasizing its non-functionality at the present time. This update ensures that users are adequately informed about the status of Pulsar and can make informed decisions regarding its use.

### CODE REVIEW

File: versioned_docs/version-latest/farming-&-staking/farming/pulsar/pulsar-tips.mdx [hunk 1]
```diff
@@ -15,8 +15,18 @@ import TabItem from '@theme/TabItem';
...
+
+:::
+
 Welcome to the Additional Tips section! Whether you're a seasoned farmer or just starting out with the Subspace Network, these tips and tricks are designed to enhance your experience and efficiency. Here, we delve into practical advice and lesser-known techniques to help you fine-tune your farming setup and navigate common challenges with ease. From configuring your environment to managing background processes, these insights are tailored to ensure your Farmer operates smoothly and effectively. Let's dive in and explore how you can get the most out of your Subspace Network Farmer.
 
+:::caution Pulsar Release Broken
+The Pulsar application is experiencing functional issues when used in conjunction with Gemini 3g. We strongly advise users to switch to the Advanced CLI method for effective farming on the Subspace Network during this period. Our team is diligently working to resolve these compatibility problems and restore full functionality to Pulsar at the earliest opportunity.
```
Comment (EmilFattakhov) [inline versioned_docs/version-latest/farming-&-staking/farming/pulsar/pulsar-tips.mdx:27]:
```suggestion
The Pulsar application is experiencing functional issues when used in conjunction with Gemini 3g. We strongly advise users to switch to the [Advanced CLI](../advanced-cli/cli-prerequisites) method for effective farming on the Subspace Network during this period. Our team is diligently working to resolve these compatibility problems and restore full functionality to Pulsar at the earliest opportunity.
```

**Review by EmilFattakhov — APPROVED** [review]
LGTM!

**Comment by jim-counter** [comment]
@ImmaZoni @EmilFattakhov Pulsar is back to being available now. Looks like this PR is fit to just revert. We will await community feedback before doing so. Thanks.

**Comment by EmilFattakhov** [comment]
@jim-counter, sounds good, noted.

**Pull request merged**